Library/Homebrew/cli/parser.rb

```diff
@@ -9,15 +9,30 @@
 require "set"
 require "utils/tty"
 
-COMMAND_DESC_WIDTH = 80
-OPTION_DESC_WIDTH = 45
-HIDDEN_DESC_PLACEHOLDER = "@@HIDDEN@@"
-
 module Homebrew
   module CLI
     class Parser
-      attr_reader :processed_options, :hide_from_man_page, :named_args_type
+      # rubocop:disable Style/MutableConstant
```

Comment:
```suggestion
      # Some comment explaining why
      # rubocop:disable Style/MutableConstant
```

Reply:
@dduugg Nudge on this one!

Reply:
Probably the same reason as this other disable. It shows up ~10 times in the codebase. Unfortunately there doesn't seem to be any real progress on the issue in 4 years.

https://github.com/Homebrew/brew/blob/5517894523620d2000f96013aa770b51dd73effc/Library/Homebrew/PATH.rb#L13-L14

Reply:
Thanks @apainintheneck!

```suggestion
      # FIXME: Enable cop again when https://github.com/sorbet/sorbet/issues/3532 is fixed.
      # rubocop:disable Style/MutableConstant
```